bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/Utils.java

```diff
@@ -58,7 +58,7 @@ public static String getURL(@NotNull SlingHttpServletRequest request, @NotNull P
     @NotNull
     public static String getURL(@NotNull SlingHttpServletRequest request, @NotNull Page page) {
         String vanityURL = page.getVanityUrl();
-        return StringUtils.isEmpty(vanityURL) ? request.getContextPath() + page.getPath() + ".html" : request.getContextPath() + vanityURL;
+        return StringUtils.isEmpty(vanityURL) ? ( request.getContextPath() + page.getPath() + ".html" ): (request.getContextPath() + vanityURL);
```

Comment:
I'd remove the space after "(" and before ")".

Reply:
removed spaces in later commit.